templates/zerver/help/switching-between-organizations.md

```diff
@@ -4,6 +4,12 @@ This article assumes you've [logged in](/help/logging-in) to each organization a
 
 {start_tabs}
 
+{tab|web}
+
+You can log in to multiple Zulip organizations by opening multiple tabs, and
+logging into one Zulip organization in each tab. To switch Zulip organizations,
+just switch tabs.
+
 {tab|desktop}
 
 1. Open the **left sidebar** (`Ctrl` + `Shift` + `s`).
```

Comment:
Let's clarify that you only have to do this if the left sidebar isn't open to begin with.